src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1030,65 +1030,65 @@ private AdsDataProvider() {
+                .put("lineItemId", 269658972)
                 .put("src", "gpt")
                 .build()
         },
         {
-            new Page("adtest", "SyntheticTests/INCONTENT_LEADERBOARD/468x60"),
+            new Page("project43", "SyntheticTests/INCONTENT_LEADERBOARD/468x60"),
             "InstantGlobals.wgAdDriverIncontentLeaderboardSlotCountries=[XX]",
             ImmutableMap.<String, Object>builder()
                 .put("slotName", AdsContent.INCONTENT_LEADERBOARD)
                 .put("slotSize", new Dimension(468, 60))
-                .put("lineItemId", 241472412)
+                .put("lineItemId", 269666292)
                 .put("src", "gpt")
                 .build()
         },
         {
-            new Page("adtest", "SyntheticTests/INCONTENT_LEADERBOARD/300x250"),
+            new Page("project43", "SyntheticTests/INCONTENT_LEADERBOARD/300x250"),
             "InstantGlobals.wgAdDriverIncontentLeaderboardSlotCountries=[XX]",
             ImmutableMap.<String, Object>builder()
                 .put("slotName", AdsContent.INCONTENT_LEADERBOARD)
                 .put("slotSize", new Dimension(300, 250))
-                .put("lineItemId", 239961132)
+                .put("lineItemId", 269672052)
                 .put("src", "gpt")
                 .build()
         },
         {
-            new Page("adtest", "SyntheticTests/Oasis/FloatingMedrecOnLongPage"),
+            new Page("project43", "SyntheticTests/Oasis/FloatingMedrecOnLongPage"),
             "",
             ImmutableMap.<String, Object>builder()
                 .put("slotName", AdsContent.FLOATING_MEDREC)
                 .put("slotSize", new Dimension(300, 250))
-                .put("lineItemId", 223909212)
+                .put("lineItemId", 269679732)
                 .put("src", "gpt")
                 .build()
         },
         {
-            new Page("adtest", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/300x600"),
+            new Page("project43", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/300x600"),
             "",
             ImmutableMap.<String, Object>builder()
                 .put("slotName", AdsContent.FLOATING_MEDREC)
                 .put("slotSize", new Dimension(300, 600))
-                .put("lineItemId", 223968492)
+                .put("lineItemId", 270230292)
                 .put("src", "gpt")
                 .build()
         },
         {
-            new Page("adtest", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/OneSkyscraper"),
+            new Page("project43", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/OneSkyscraper"),
             "",
             ImmutableMap.<String, Object>builder()
                 .put("slotName", AdsContent.FLOATING_MEDREC)
                 .put("slotSize", new Dimension(300, 600))
-                .put("lineItemId", 223968492)
+                .put("lineItemId", 270230292)
                 .put("src", "gpt")
                 .build()
-        },
+        }/*,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ATrailingCommentCheck)